conf/rubocop_without_rspec.yml

```diff
@@ -1,3 +1,7 @@
+require:
+  - rubocop-performance
+  - rubocop-rails
```

Comment:
Why did we need this change? If we do need it, should the rubocop-rails require go in rubocop_rails.yml?

Reply:
The need for this is that otherwise we get these errors:
![image](https://user-images.githubusercontent.com/8846579/123322843-a9dc5a80-d52c-11eb-9eab-73986dc2f774.png)

As for moving `require: rubocop-rails` to `rubocop_rails.yml`, the issue is that this file has the following lines:
https://github.com/salsify/salsify_rubocop/blob/8dd23695141a9ae57c9f672a4bec9ba8edb46862/conf/rubocop_without_rspec.yml#L192-L193

Reply:
I guess this gem depends on [rubocop-rails](https://github.com/rubocop/rubocop-rails) even if the project isn't using Rails so this should be innocuous.